com.unity.renderstreaming/Documentation~/input.md

```diff
@@ -106,3 +106,51 @@ You can control the UnityUI placed on the game scene from browser.
 > [!NOTE]
 > If the Unity application is in Background, even if the `RunInBackground` checkbox is On, you can not control the UnityUI from browser. This issue will be fixed in `com.unity.inputsystem`.
 
+
+
+### Workaround for using UnityUI on RunInBackground
+
+If use UnityUI in the background, follow the steps below.
+
+1. Update `com.unity.inputsystem` to `1.1.0-preview2` on PackageManager.
+
+2. In ProjectSetting->Player->OtherSetting, check `Allow unsafe code`.
+
+3. Replace the floowing Component with `EventSystem` on the Scene.
```

Comment:
~Replace the floowing Component with `EventSystem` on the Scene.~
Replace `EventSystem` in the Scene with the following component.